app/components/Wrapper.tsx

```diff
@@ -6,39 +6,41 @@ import { useAccount, useDisconnect, useNetwork, useSwitchNetwork } from "wagmi";
 import { Buffer } from "buffer";
 import Modal from "./Modal";
 import RewardsHeader from "./RewardsHeader";
+import { filteredNetwork } from "~/utils/helpers";
 
 export default function Wrapper({ children, network }: { children?: ReactElement; network: string }) {
   let link: string;
   let linkText: string;
-  
+
   const [isOpen, setIsOpen] = useState(false);
   const location = useLocation();
   const { address, connector } = useAccount();
   const { disconnect } = useDisconnect();
   const { chain, chains } = useNetwork();
-  
-  const primaryChainObj = chains.filter(chainObj => {
-    return chainObj.name?.toLowerCase() === network
-  })[0]
 
-  const { switchNetwork } = useSwitchNetwork({ 
+  const primaryChainObj = chains.filter((chainObj) => {
+    return chainObj.name?.toLowerCase() === filteredNetwork(network);
```

Comment:
Cannot believe this was the solution. Great catch as I was about to tear my hair out over this

Reply:
I hear ya if you work at something too long you usually miss the smallest detail